src/types/algod.ts

```diff
@@ -1,145 +1,20 @@
-   * An empty string indicates the transaction wasn't kicked out of this node's txpool due to an error. */
-  'pool-error': string
-  /**
-   * Rewards in µALGOs applied to the receiver account.
-   */
-  'receiver-rewards'?: number
-  /**
-   * Rewards in µALGOs applied to the sender account.
-   */
-  'sender-rewards'?: number
-  /**
-   * The raw signed transaction.
-   */
-  txn: EncodedSignedTransaction
-}
-
-/** Represents a TEAL value delta https://developer.algorand.org/docs/rest-apis/algod/v2/#evaldelta */
-export type EvalDelta =
-  | {
-      /**
-       * Value `1` refers to **bytes**, value `2` refers to **uint**
-       */
-      action: 1
-      /**
-       * Bytes value.
-       */
-      bytes: string
-    }
-  | {
-      /**
-       * Value `1` refers to **bytes**, value `2` refers to **uint**
-       */
-      action: 2
-
-      /**
-       * Uint value.
-       */
-      uint: number | bigint
-    }
-
-/** The response from the application API https://developer.algorand.org/docs/rest-apis/algod/v2/#get-v2applicationsapplication-id */
-export interface ApplicationResponse {
-  id: number
-  params: ApplicationParams
-}
-
-/** Stores the global information associated with an application https://developer.algorand.org/docs/rest-apis/algod/v2/#applicationparams */
-export interface ApplicationParams {
-  /** Address of the account that created the app */
-  creator: string
-  /** Base64 encoded TEAL approval program */
-  'approval-program': string
-  /** Base64 encoded TEAL clear state program */
-  'clear-state-program': string
-  /** The amount of extra program pages available to this app. */
-  'extra-program-pages'?: number
-  /** Current global state values */
-  'global-state'?: { key: string; value: TealValue }[]
-  /** Global state schema */
-  'global-state-schema'?: ApplicationStateSchema
-  /** Local state schema */
-  'local-state-schema'?: ApplicationStateSchema
-}
-
-/**
- * Represents a TEAL value https://developer.algorand.org/docs/rest-apis/algod/v2/#tealvalue
- */
-export type TealValue =
-  | {
-      /**
-       * (tt) value type. Value `1` refers to **bytes**, value `2` refers to **uint**
-       */
-      type: 1
-      /**
-       * (tb) bytes value.
-       */
-      bytes: string
-    }
-  | {
-      /**
-       * (tt) value type. Value `1` refers to **bytes**, value `2` refers to **uint**
-       */
-      type: 2
-
-      /**
-       * (ui) uint value.
-       */
-      uint: number | bigint
-    }
-
-/** Specifies maximums on the number of each type that may be stored https://developer.algorand.org/docs/rest-apis/algod/v2/#applicationstateschema */
-export interface ApplicationStateSchema {
-  /** [nbs] num of byte slices */
-  'num-byte-slice': number
-  /** [nui] num of uints */
-  'num-uint': number
-}
+import { modelsv2 } from 'algosdk'
+
+export const PendingTransactionResponse = modelsv2.PendingTransactionResponse
```

Comment:
@tristanmenzel worth keeping the jsdoc comment with the link to the documentation or should we even remove this file altogether? Maybe exposing these types is still handy because the location in the orginal is a little odd...

Reply:
we could probably drop the file all together and just replace usages with the namespaced version